resources/web/docs_js/components/console_widget.jsx

```diff
@@ -33,23 +33,25 @@ export class _ConsoleForm extends Component {
 
     return (
       <form>
+         <p><strong>Want to use the Kibana Console?</strong></p>
         <label for="url">{props.langStrings(props.url_label)}</label>
         <input
           id="url"
           type="text"
           value={getValueFromState('url')}
           onInput={linkState(this, getFieldName('url'))}
         />
-
-        <label for="curl_host">curl {props.langStrings('host')}</label>
+        <p></p>
+        <p><strong>Want to use curl? (basic auth)</strong></p>
```

Comment:
```suggestion
        <p><strong>Copy as curl</strong></p>
        <p>Include your Elasticsearch settings in curl code examples.<p>
```

Reply:
I don't _love_ this.

Reply:
Me neither, I've kept the settings page changes minimal, but I think it's pretty clear now